src/controllers/auth.ts

```diff
@@ -147,45 +159,45 @@ export const authenticateToken = (req: Request, res: Response, next: NextFunctio
 
 export const authenticateRefreshToken = (req: Request, res: Response, next: NextFunction) => {
     const token = req.cookies.jid;
-    if (token == null) return res.status(401).json({ message: "JWT Refresh Token is missing, access denied" })
-    let payload: any = null
+    if (token == null) return res.status(401).json({ message: "Invalid refresh Token" })
     if (process.env.REFRESH_TOKEN_SECRET) {
         const refreshTokenSecret: string = process.env.REFRESH_TOKEN_SECRET
         try {
-            payload = verify(token, refreshTokenSecret)
+            let payload: any = verify(token, refreshTokenSecret)
```

Comment:
any <-- I will just mark them, so you can decide if you put them consciously there or if you just have forgotten to remove them.